### PR TITLE
DataTable: Added functionality to update a datatable or add new data

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

File: src/plugins/tables/tables.js [hunk 1]
```diff
@@ -147,6 +153,19 @@ var componentName = "wb-tables",
 // Bind the init event of the plugin
 $document.on( "timerpoke.wb " + initEvent, selector, init );
 
+// Bind the update event of the plugin
+$document.on( updateEvent, selector, function( event ) {
+	var $elm = $( event.currentTarget );
+
+	$elm.DataTable().destroy();
+
+	if ( event.newData ) {
```
Comment (andre-gh) [inline src/plugins/tables/tables.js]: newData is a parameter to clear the table? Why?

Comment (LaurentGoderre) [inline src/plugins/tables/tables.js]: For web application this is a common use case....

Comment (andre-gh) [inline src/plugins/tables/tables.js]: Right, but how do I actually add new data using your update?

Comment (LaurentGoderre) [inline src/plugins/tables/tables.js]: If you send data using `data-wb-tables` and then want to modify this data and add new one, you would use this.

**Comment by shawnthompson** [comment]
cc/ @masterbee

**Pull request closed without merging**